### ISSUE TITLE
ArrayCamera rotation support (including OrbitControls.js)

### ISSUE THREAD

**Issue by alxdncn** [mrdoob/three.js#15107]
It would be great to be able to rotate an array camera as you would a regular camera, i.e. either accessing it's .rotation property or passing the camera into an orbit controls script. While this isn't particularly valuable for VR applications, any other type of application that leverages the optimization possible through ArrayCamera has to do a lot of workarounds to make rotation function properly.

**Comment by alxdncn** [comment]
I was able to solve this by copying the main camera's rotation to each of the subcameras every frame, but that seems a little circuitous.

**Comment by Mugen87** [comment]
@mrdoob We have a few issues regarding `ArrayCamera` where people expect this class behaves like other camera classes.

I think `ArrayCamera` should focus on what it does in context of XR. Meaning providing the possibility for single-pass rendering. So stuff like updating render lists and states or shadows is done only once when using `ArrayCamera`.

Have you considered to make the class internal and not expose it to users? Otherwise I would suggest to change its name and not derive it from `PerspectiveCamera`. Instead the class could have a camera property of type `PerspectiveCamera`.

**Comment by Mugen87** [comment]
I don't think we have to do anything to support the OPs feature request. `ArrayCamera` can actually operate in two ways:

- Decoupled like in `webgl_camera_array`: Sub cameras are independent of the array camera's transform. 
- Inherited (the OP's request): Sub cameras as _children_ of the main array camera so they inherit the transform via the parent-child relationship. It's still possible to apply offsets to the sub cameras.

An other design concerns regarding `ArrayCamera` like in https://github.com/mrdoob/three.js/issues/15107#issuecomment-812456007 are not further relevant to this issue.
